docs/checkedc/Test-Baselines.md

```diff
@@ -52,6 +89,16 @@ Here is the current base line for testing LLVM + clang on all targets (check-all
            Unexpected Failures: 6
 ```
 
+## Delta between the master branch and baseline branch
+
+We have added tests for Checked C to the master branch.  These tests are specific to clang, such as tests of
+compiler internals or the driver.  We currently expect the master branch to pass the following additional tests
+in these configurations.
+
+- For just clang with the x86 target (using the check-clang project), 3 additional `Expected Passes` tests.
+- For testing LLVM + clang on x86 (check-all), 8 additional `Expected Passes` tests
```

Comment:
I saw your updated instructions for keeping this .md file up to date, but I just get the sense that it's going to get out-of-sync.  (And if it stays in sync, I'm wondering if it's going to be unnecessary developer burden for each change that adds a new test.)

I know it's important to keep in agreement on which failures are expected (because they show up in the base line) and agree we should keep that updated--but do we need to update the counts of tests we've added?  I'm not sure how much we'll get from that...